subsys/bluetooth/host/att.c

```diff
@@ -178,36 +188,97 @@ static int chan_send(struct bt_att_chan *chan, struct net_buf *buf,
 		}
 	}
 
+	chan->sent = cb ? cb : chan_cb(buf);
+
 	return bt_l2cap_send_cb(chan->att->conn, BT_L2CAP_CID_ATT, buf,
-				att_sent, &chan->chan.chan);
+				att_cb(chan->sent), &chan->chan.chan);
+}
+
+static int process_queue(struct bt_att_chan *chan, struct k_fifo *queue)
+{
+	struct net_buf *buf;
+	int err;
+
+	while ((buf = net_buf_get(queue, K_NO_WAIT))) {
```

Comment:
`while` with always one iteration? change to `if`?

Reply:
Why only one iteration? This will loop until `buf != NULL`? which btw, why `K_NO_WAIT` in this case and busy-loop like that?

Reply:
That indeed can be converted to if statement, I will change that.

Reply:
Done.